### ISSUE TITLE
Some examples do not work

### ISSUE THREAD

**Issue by AlexEne** [Grokmoo/thyme#31]
The code here: https://docs.rs/thyme/0.5.0/thyme/  mentions:
```rust
    ui.child("custom_widget")
    .align(Align::BotRight)
    .layout(Layout::Vertical)
    .children(|ui| {
      for i in 1..10 {
        ui.label("label", format!("Row #{}", i));
      }
    });
```

This errors out as align is not found in `WidgetState`

**Comment by AlexEne** [comment]
For reference, I'm trying to re-create this window layout in thyme:
![image](https://user-images.githubusercontent.com/5849037/102987093-c0681180-4509-11eb-9acf-12404bcf9e67.png)

**Comment by Grokmoo** [comment]
You should be able to just insert

`use thyme::{Align, Layout};`

before that and it should work.  You shouldn't have any issue recreating that layout with Thyme either in the code or using the YAML attributes.

**Comment by AlexEne** [comment]
The problem is that this line `ui.child("custom_widget")` returns [`WidgetState`](https://github.com/Grokmoo/thyme/blob/master/src/recipes.rs#L20) and that's an [enum](https://github.com/Grokmoo/thyme/blob/master/src/widget.rs#L197) that doesn't have `align` or the other methods available.

It may seem that the intent was that `add_child` should return a `Widget`?

**Comment by Grokmoo** [comment]
Ah!  Sorry about that, you should replace `ui.child(` with `ui.start(`

the `child` method is a shorthand that immediately finishes building the widget without any extra behavior.  I'd recommend you check out the demo example.

**Comment by AlexEne** [comment]
Oh, ok I can raise a PR to fix the example and maybe enable doc tests.

**Comment by Grokmoo** [comment]
Fixed in 366ddbe28b39cab74cf7d7fd31f599ba6ea2450e
